### PR TITLE
Add historical offline inboxing idea

### PR DESCRIPTION
**Pull request by oskarth**
I was curious to see what historical ideas would look like, and it might be useful to have these as well. At least a subset of things that were completed/aborted - probably a lot of hidden lessons learned and contexts etc there.

### CODE REVIEW

File: ideas/1-offline-inboxing.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- Running wnodes in cluster and making it possible for others to run them
+- Giving feedback in chat about state of messages
+- Using multiple wnodes
+- Cryptoeconomics to incentive running of nodes
+- Basic reputation system to signal quality of service
+
+Goal Date:
+Description:
+
+## Supporting Role Communication
```
Comment (oskarth) [inline ideas/1-offline-inboxing.md]: Missing success metric and exit criteria. Not part of requirements for ideas at the time. Probably useful to add something ad-hoc here.

File: ideas/1-offline-inboxing.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+TBD.
+
+## Post-Mortem
+
+TBD.
```
Comment (oskarth) [inline ideas/1-offline-inboxing.md]: This is in a document somewhere

File: ideas/1-offline-inboxing.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- How communicate to the sender that a message was ACKed by a mail box but not by end user?
+- What about if a user wants to use multiple mailboxes?
+- Where can a user get an overview of the mail server / contracts they are using?
+- How will ongoing payment signing work?
+- How do we communicate a certain wnode is dead or can't be trusted for some reason?
+- How do we communicate capacity of a wnode to ensure quality of service?
+- How does a wnode operator accept or deny a request to act as mail server? How is this communicated to end user?
+- Initially a wnode will probably just be a CLI thing. How do we surface options like how much compensation we want? And what about when this can be run through Status Desktop? How do you decide how much capacity you have as a wnode?
+
+#### Iteration 1..N
```
Comment (oskarth) [inline ideas/1-offline-inboxing.md]: Missing iterations updates, should be somewhere

**Comment by oskarth** [comment]
Requesting reviews based on people who worked on it and one person (@martinklepsch) who didn't. See if you feel like it this is:

(a) useful
(b) captures essence of https://github.com/status-im/ideas/issues/1

**Comment by adambabik** [comment]
I like your comment @dmitryn! If we had more metadata attached to every idea, we could generate pretty interesting stats. The thing is that each swarm would need to operate in a more structured way which kind of stands in contrast to the whole swarm concept (personally, I have nothing against more structured swarms, I even support that :D).

**Comment by martinklepsch** [comment]
I think it would take me too long to understand the essence of #1 to judge whether this really captures it. One thing that might be worth considering is a one-paragraph summary of what the swarm attempted to do, what it achieved / why it failed.

Preserving this kind of information over the long term definitely seems useful. I think a lot of context that accrues over the lifecycle of a swarm might get lost but we can slowly figure our if/how we can preserve that. Linking to related issues/project boards as @dmitryn suggested could be a good start for sure.

**Comment by oskarth** [comment]
Good idea @dmitryn!

One problem with org project boards is that they aren't open (accessible to everyone), so we probably should move away from them. I did however apply the label "1-offline-inboxing" to all issues in the two project boards, and anyone can seem them here: https://github.com/search?q=org%3Astatus-im+label%3A1-offline-inboxing (except for the few done in status-cluster private repo). I think this would be a great starting point for more detailed analytics around contributors etc too, since this can easily be queried through GH API.

WDPT about using this label system for all swarms going forward?

**Comment by oskarth** [comment]
> I think it would take me too long to understand the essence of #1 to judge whether this really captures it. One thing that might be worth considering is a one-paragraph summary of what the swarm attempted to do, what it achieved / why it failed.

I think the first part is captured in summary, and second part in post mortem? Maybe it's a matter of writing better post-mortems in the future that are more digestible. Good feedback!

Also added link to issues solved.

**Pull request merged**